apps/barista-design-system/src/components/ba-decision-graph/ba-decision-graph-node/ba-decision-graph-node-navigation/ba-decision-graph-node-navigation.scss

```diff
@@ -0,0 +1,61 @@
+@import 'colors';
+@import '../../../../styles/global/variables';
+
+.ba-uxd-navigation-wrapper {
+  width: 100%;
+  margin-top: 32px;
+}
+
+@media only screen and (min-width: $screen-lg) {
```

Comment:
do we need the only screen? I think you can remove it because on small printed outlets it should behave the same.